apps/admin-gui/src/app/vos/pages/group-detail-page/group-overview/group-overview.component.ts

```diff
@@ -52,6 +58,23 @@ export class GroupOverviewComponent implements OnInit {
           this.initNavItems();
           this.loading = false;
         }
+
+        this.groupService.getGroupMembersCount(this.groupId).subscribe(count => {
+          this.membersCountsByVoStatus.set('members', count);
+          this.membersCountsByGroupStatus.set('members', count);
+        }, () => this.loading = false);
+
+        this.groupService.getGroupMembersCountsByVoStatus(this.groupId).subscribe(stats => {
+          Object.entries(stats).forEach(([status, count]) =>
+            this.membersCountsByVoStatus.set(status.toLowerCase(), count)
+          );
+        }, () => this.loading = false);
+
+        this.groupService.getGroupMembersCountsByGroupStatus(this.groupId).subscribe(stats => {
+          Object.entries(stats).forEach(([status, count]) =>
+            this.membersCountsByGroupStatus.set(status.toLowerCase(), count)
+          );
+        }, () => this.loading = false);
```

Comment:
we usually want to make this asynchronous operations chained - so that on the end of the chain is loading = false and we can say that every information is loaded
but this section will be redesigned in this sprint